README.md

```diff
@@ -1,8 +1,8 @@
 # Kueue
 
-Kueue is a set of APIs and controller for [job](docs/concepts/workload.md)
-[queueing](docs/concepts#queueing). It is a job-level manager that decides when
-a job should be [admitted](docs/concepts#admission) to start (as in pods can be
+Kueue is a set of APIs and controller for [job](https://kueue.sigs.k8s.io/docs/concepts/workload)
```

Comment:
Could you update this section to match https://kueue.sigs.k8s.io/docs/overview/?

Although maybe instead of repeating the "Why use Kueue" section, we can just point to the page.

Reply:
> Although maybe instead of repeating the "Why use Kueue" section, we can just point to the page.

Sounds good.

Reply:
Done.